src/include/pbs_nodes.h

```diff
@@ -265,7 +266,7 @@ struct	pbsnode {
 	long			 nd_nsnfree;	/* number of VPs free */
 	long			 nd_ncpus;	/* number of phy cpus on node */
 	short			 nd_written;	/* written to nodes file */
-	unsigned short		 nd_state;	/* state of node */
+	unsigned long		 nd_state;	/* state of node */
```

Comment:
OK I changed my mind due to one thing.  The attribute at_val structure does not have an int.  It has a short, long, or long long.  If we're stuck using a long, we might as well go all in.  I tracked down all the places the INUSE_* flags are used and modified their usage to be a unsigned long.  I wrote a nice comment on top of the defines for the INUSE_* macros that says if more than 32 flag bits are required, the database schema will need to be updated.